src/digitalmarket/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
```

Comment:
This code is not related to back end. It is template related. 
It is OK to leave it here, just wanted to mention it, for consistency